census/README.md

```diff
@@ -233,6 +227,22 @@ gcloud ml-engine jobs submit training $JOB_NAME \
                                     --eval-steps 100
 ```
 
+## Accuracy and Output
+The output will conclude by reporting the accuracy of the trained model on the evaluation dataset. For the default number of training steps, the accuracy should be close to `80%`.
+
+## Tensorboard
+Run [Tensorboard](https://www.tensorflow.org/get_started/summaries_and_tensorboard) to inspect the details about the graph.
+
+If you saved the model locally at `$OUTPUT_DIR`, run:
+```
+tensorboard --logdir=$OUTPUT_DIR
+```
+
+If you saved the model on Google Cloud Storage at `$GCS_JOB_DIR`, run:
+```
+tensorboard --logdir=$GCS_JOB_DIR
+```
+
 # Hyperparameter Tuning
 Cloud ML Engine allows you to perform Hyperparameter tuning to find out the
```

Comment:
Please fix this (long link is displayed)

Cloud ML Engine allows you to perform Hyperparameter tuning to find out the most optimal hyperparameters. See [Overview of Hyperparameter Tuning] (https://cloud.google.com/ml/docs/concepts/hyperparameter-tuning-overview) for more details.